package-examples/helloworld-kustomize/README.md

```diff
@@ -0,0 +1,67 @@
+# helloworld-kustomize
+
+This is an example of a kpt package that has a kustomize
+patch in it.
+
+## Steps
+
+This is a sample workflow where 
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Configure functions](#configure-functions)
+4. [Render the declared values](#render-the-declared-values)
+5. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+    $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld-kustomize
+
+      fetching package /package-examples/helloworld-kustomize from https://github.com/GoogleContainerTools/kpt to helloworld-kustomize
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+    $ kpt pkg tree helloworld-kustomize/
+
+      PKG: helloworld-kustomize
+      ├── [Kptfile]  Kptfile helloworld-kustomize
+      ├── patches
+      │   └── [patch.yaml]  Deployment helloworld-gke
+      └── resources
+          ├── [deploy.yaml]  Deployment helloworld-gke
+          └── [service.yaml]  Service helloworld-gke
+
+### Configure functions
+
+The package contains a function pipeline in the `Kptfile` which has
+one `apply-setter` function.  The `apply-setter` function allows you to
+set a simple value throughout the package configuration.  In this case the
+value of the setter goes into the `patch.yaml`.  You can set the target
+environment variable to a value different of your choice (different
+than `foobar`)
+
+    pipeline:
+      mutators:
+        - image: gcr.io/kpt-fn/apply-setters:unstable
+          configMap:
+            target: foobar
+
+
+### Render the declared values
+
+Render the changes in the hydration pipeline by using `kpt fn render` command:
+
+    $ kpt fn render helloworld-kustomize/
+
+### Apply the package
+
+Initialize the inventory object:
+
+    $ kubectl apply -k helloworld-kustomize
```

Comment:
So we recommend using `kubectl apply` directly here ? I agree that it is the simplest thing to do here but this would add friction for Day n workflow. What if one resource is deleted in the upstream?

I think we should follow consistency here. How about `kustomize build | kpt live apply`? I am sure there could be some friction with that flow but we should try and explore that route. I think this is an important example.

I am fine with this flow as well if you are convinced.

Reply:
I think we need to rethink what our kpt+kustomize sample should be.  I am not convinced it should be this sample :) this is just a migration of content.